tests/tests.schema

```diff
@@ -10837,7 +10837,7 @@
           }
         },
         "priority": {
-          "$ref": "#/definitions/integerExpression",
+          "$ref": "#/definitions/numberExpression",
           "title": "Priority",
```

Comment:
Where did all of these schema changes come from?  They don't look to be added by your changes and look related to switching priority from integer to float.  Given our tests, I don't know how these kinds of changes can get checked-in without the tests getting updated.  It likely means someone did not run the tests locally, but I thought then the test would fail on the server.